mesheryctl/internal/cli/root/system/context/view.go

```diff
@@ -28,9 +28,12 @@ var viewContextCmd = &cobra.Command{
 			log.Print(getYAML(configuration.Contexts))
 			return nil
 		}
-
+		if len(args) != 0 {
+			context = args[0]
+		}
 		if context == "" {
 			context = viper.GetString("current-context")
+			log.Printf("***This is your Current context***")
```

Comment:
We do not need a log like this. The idea is that `mesheryctl system context view` will display the current context. And if the user wants to specifically view another context, he will use the `mesheryctl system context view <context name>`.

So, IMO we do not really need to print this.